packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -97,6 +97,9 @@ export class Runner {
             pageScanResult.run = this.createRunResult('completed');
             pageScanResult.scanResult = this.getScanStatus(axeScanResults);
             pageScanResult.reports = [await this.saveScanReport(axeScanResults)];
+            if (axeScanResults.redirectedToUrl !== undefined) {
```

Comment:
If the default value of `pageScanResult.redirectedToUrl` is `undefined` is this check necessary? Or are we trying to preserve an earlier set value here?

Reply:
Yes, if no value is set then that field doesn't get written